.changeset/proud-ads-relate.md

```diff
@@ -0,0 +1,20 @@
+---
+"@hopper-ui/icons-react16": major
+---
+
+**Breaking Change**: Updated the icons variables to be more consistent with the other components.
+
+Will need to update the style sheets to use the new CSS variables.
+
+For example, they will need to replace:
+
+| Old variable | New variable |
+| ------------ | ------------ |
+| `--hop-richicon-placeholder-surface-color` | `--hop-Richicon-placeholder-shadow` |
```

Comment:
This might be too late but shouldn't this be RichIcon?

Reply:
plus one has a capital R and the other doesn't, is this normal?

Reply:
The first variable with the lowercase r is the old one, which has been replaced by the one with an R